fastchat/serve/openai_api_server.py

```diff
@@ -90,11 +91,25 @@ async def check_model(request) -> Optional[JSONResponse]:
             )
     return ret
 
-
+@app.post("/v1/chat/completions")
```

Comment:
This is already a registration for this endpoint at https://github.com/lm-sys/FastChat/blob/f5c3f255578e2b0bf4ed252ea4bb4612a87c4d5c/fastchat/serve/openai_api_server.py#L287

Can we merge this modification to the existing endpoint?

Reply:
solved